ts/features/pn/components/TimelineListItem.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useMemo } from "react";
+import { Dimensions, View } from "react-native";
+import {
+  Alert,
+  IOVisualCostants,
+  ListItemAction
+} from "@pagopa/io-app-design-system";
+import { useIOBottomSheetModal } from "../../../utils/hooks/bottomSheet";
+import I18n from "../../../i18n";
+import { NotificationStatusHistory } from "../../../../definitions/pn/NotificationStatusHistory";
+import { formatDateAsDay, formatDateAsMonth } from "../../../utils/dates";
+import { localeDateFormat } from "../../../utils/locale";
+import {
+  getNotificationStatusInfo,
+  notificationStatusToTimelineStatus
+} from "../utils";
+import { trackPNTimelineExternal } from "../analytics";
+import { handleItemOnPress } from "../../../utils/url";
+import { useIOSelector } from "../../../store/hooks";
+import { pnFrontendUrlSelector } from "../../../store/reducers/backendStatus";
+import { Timeline, TimelineItemProps } from "./Timeline";
+
+const topBottomSheetMargin = 122;
+const timelineBottomMargin = 292;
+const timelineItemHeight = 70;
```

Comment:
Ui-wise it seems to work as intended, but may I suggest the usage of `useIOBottomSheetAutoresizableModal`?

Reply:
`useIOBottomSheetAutoresizableModal` does not work and it is buggy, as you shown before